File: python-starter/functions/hello.py
from nitric.resources import api
from nitric.application import Nitric
from nitric.faas import HttpContext

helloApi = api("main")


@helloApi.get("/hello/:name")
async def hello_world(ctx: HttpContext):
    name = ctx.req.params['name']

    ctx.res.body = f"Hello {name}"

Nitric.run()

File: csharp-starter/functions/Hello.cs
using Nitric.Sdk;

var api = Nitric.Sdk.Nitric.Api("main");

api.Get("/hello/:name", context =>
{
    var name = context.Req.PathParams["name"];

    context.Res.Text($"Hello {name}!");

    return context;
});

Nitric.Sdk.Nitric.Run();